server/audit_math/bravo.py

```diff
@@ -342,6 +344,12 @@ def get_sample_size(
 
     samples: Dict = {}
 
+    if round_sizes:
+        num_sampled = sum([round_sizes.get(rnd, 0) for rnd in round_sizes])
+        # If we've already sampled all the ballots, we should never be here
```

Comment:
if we should never be here, shouldn't we just raise an exception?

Reply:
I can do that!